src/rules/no-legacy-imports.ts

```diff
@@ -43,7 +44,7 @@ const rule: Rule.RuleModule = {
       'queryAssignedNodes',
       'queryAsync'
     ];
-    const movedSources: Array<{from: RegExp; to: string}> = [
+    const movedSources: Array<{ from: RegExp; to: string }> = [
```

Comment:
did prettier do this? just wanna be sure we're not manually fixing up formatting. if it didn't, and you did it manually, maybe run the formatter again just to be sure nothing else needs formatting?

Reply:
tbh it'd be better to use prettier itself to check the formatting in CI (it has a `--check` flag or something). then have any conflicting formatting rules turned off (doesn't necessarily need the prettier plugin for that).

though this looks good now at least